src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java

```diff
@@ -265,39 +265,39 @@ public void testDiagramsAndGraphics() throws Exception {
         UUID modificationNodeUuid = modificationNode1.getId();
 
         //get the voltage level diagram svg
-        mockMvc.perform(get("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network/voltage-levels/{voltageLevelId}/svg?useName=false",
+        mockMvc.perform(get("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network/voltage-levels/{voltageLevelId}/svg?useName=false&language=en",
```

Comment:
Add also a test without providing the 'language' parameter (because it is an optional one, with a default value)